### DISCUSSION TITLE
Can't re-verify my student status.

### DISCUSSION THREAD

**Discussion by sam-sampreeth** [community/community#51342] [GitHub Education]
I recieved a mail today saying I have to re-verify my student status. I am being shown the following error:

```
We require applicants of Dayananda Sagar University to use one of these school-issued email addresses to apply:

dsu.edu.in

dayanandasagar.edu
Please select a different email address.
```

My college has not provided me with a student email ID. Is there any other way I can verify my student status?

I didn't face any error like this while i applied for the first time.

Thank you for your assistance.

**Comment by samus-aran** [comment]
Hi @sam-sampreeth 

First off I want to thank you for sharing your experiences and that we absolutely understand your frustration. We are trying our best to continually improve our system to be as easy as possible for legitimate students everywhere to verify their academic status, while also ensuring that bad actors impersonating students don't jeopardise our ability to provide academic benefits of value.

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been associated with this school.

We would recommend to you speak to your institution regarding to acquiring an email address to proceed.

**Reply by sam-sampreeth** [reply]
Hey @samus-aran 
Thanks for replying!

> We would recommend to you speak to your institution regarding to acquiring an email address to proceed.

Sadly, my institution provides the edu.in emails only for the professors.  
I hope GitHub provides an alternative method to verify the student status.

**Reply by C-crime** [reply]
Sadly, I would ensure you that a e-file would suffice the cause of a dodge at a hearing in the near future. In front of the board...good luck.
                    -Cesar Vargas

**Comment by Aditya753b** [comment]
Yeah, I'm facing the same problem. Even my institution doesn't provide me edu mail. They're only given to professors. 
![Screenshot 2023-04-16 091202](https://user-images.githubusercontent.com/69055133/232288891-e76b41ae-6fd6-4b47-a658-6c8ce27d2719.png)
please check my application too.

**Comment by Aditya753b** [comment]
> I recieved a mail today saying I have to re-verify my student status. I am being shown the following error:
> 
> ```
> We require applicants of Dayananda Sagar University to use one of these school-issued email addresses to apply:
> 
> dsu.edu.in
> 
> dayanandasagar.edu
> Please select a different email address.
> ```
> My college has not provided me with a student email ID. Is there any other way I can verify my student status?
> 
> I didn't face any error like this while i applied for the first time.
> 
> Thank you for your assistance.

Well u can u do one thing u can go  meet ur higher staffs or deans or write a simple letter stating that u need a student email ID for educational purposes and give a reference link below the letter and tell them to atleast provide you with a maill id consisting your college id or register number including with the college mail id

**Reply by Aditya753b** [reply]
If u need any more help feel free to contact me : )
